JsonLogic/JsonLogic.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFramework>netstandard2.1</TargetFramework>
+		<TargetFrameworks>netstandard2.1;net8.0</TargetFrameworks>
```

Comment:
I thought we were dropping netstandard with this change?

Reply:
No.  Actually, because of PolySharp, I might even be able to move back to NS 2.0!  I'll check that.

We dropped support for .Net Core 3.1 because of an incompatiblity in a dependency of STJ v8, but I still want to have .Net Standard for support outside of .Net 8.

Reply:
Looks like Logic can be pushed down to .Net Standard 2.0.  All I needed was to add a deconstructor for `KeyValuePair<,>`.

But Path is using all sorts of new APIs, like a lot of what's defined on spans.